app/server/lib/DocApi.ts

```diff
@@ -518,6 +528,27 @@ export class DocWorkerApi {
       })
     );
 
+    // A GET /sql endpoint that takes a query like ?q=select+*+from+Table1
+    // Not very useful, apart from testing - see the POST endpoint for
```

Comment:
Once args are part of the query, it'd be useful, but I didn't want to invest in that (e.g. handling types) since I didn't know whether this PR would pass scrutiny. Datasette does a quite beautiful job on a similar endpoint. It would be more work to make beautiful via node-sqlite3, it'd be better with named parameters rather than just a list.

I can see that "useful" is a judgement call, I say it because any SQL interface that is string only is kind of a pain to work with and invites problems.